File: AppKitExample/src/main/java/me/grishka/appkit/example/MainActivity.java
package me.grishka.appkit.example;

import android.os.Bundle;

import me.grishka.appkit.FragmentStackActivity;
import me.grishka.appkit.example.api.PlaceholderAPIController;
import me.grishka.appkit.example.fragments.TestSimpleFragment;

public class MainActivity extends FragmentStackActivity{

	static{
		PlaceholderAPIController.getInstance();
	}

	@Override
	protected void onCreate(Bundle savedInstanceState){
		super.onCreate(savedInstanceState);
		if(savedInstanceState==null){
			showFragmentClearingBackStack(new TestSimpleFragment());
		}
	}
}

File: AppKitExample/src/main/java/me/grishka/appkit/example/MainActivity.kt
package me.grishka.appkit.example

import android.os.Bundle

import me.grishka.appkit.FragmentStackActivity
import me.grishka.appkit.example.api.PlaceholderAPIController
import me.grishka.appkit.example.fragments.TestSimpleFragment

class MainActivity : FragmentStackActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        if (savedInstanceState == null) {
            showFragmentClearingBackStack(TestSimpleFragment())
        }
    }

    companion object {

        init {
            PlaceholderAPIController.getInstance()
        }
    }
}
